keep-ui/app/settings/users-settings.tsx

```diff
@@ -39,61 +41,50 @@ export default function UsersSettings({
   const apiUrl = getApiURL();
   const { data, error, isLoading } = useSWR<User[]>(
     selectedTab === "users" ? `${apiUrl}/settings/users` : null,
-    (url) => fetcher(url, accessToken),
+    async (url) => {
+      const response = await fetcher(url, accessToken);
+      setUsers(response); // Update users state
+      return response;
+    },
```

Comment:
What is this required? "data" is the same, isn't it?

Reply:
not sure what you mean